### ISSUE TITLE
something not right on the "You have no series yet!" page? (DuckieTV DEV 0.95)

### ISSUE THREAD

**Issue by garfield69** [SchizoDuckie/DuckieTV#384]
Something is not right with the "you have no series yet!" page
![image](https://cloud.githubusercontent.com/assets/6933240/6659560/1724d40a-cbed-11e4-90fa-e966bf88835e.png)

**Comment by Js41637** [comment]
Yeah I've spoke to Duckie about this. THere are a number of problems.

1. The .adding class doesn't get added to the series-list which means the posters are missing some CSS to fix the overflow problem
2. The Import is broken
3. The trakt sidepanel doesnt open when hovering series to view info about them
4. Opening the actual sidepanel (settings, about, ect) closes the favs-list and opens the sidepanel instead of resizing

**Comment by Js41637** [comment]
5. The Top Sites should also probably be behind the favs list
6. There is a short instance when the favs list opens with no series  where the tools panel quickly changes from the normal 'Add a serie' to 'im done adding series'

**Comment by Js41637** [comment]
Most of these issues are being caused by the fact that having no series serves up a different template, I don't think it's ideal to have 2 templates for the same page (trending) when the only real difference is one shows the 'You have no series' alert. We can still put that in the current trending and just have an ngIf noFavorites. Also means any changes to the layout of trending will have to be redone in the empty-import page as well.

Also because of the way that the serieslist opens when there are no series ($state.go) it opens very fast, because the initial opening animation is skipped (serieslist.activated). I tried for a while to get it to activate it before the state.go however no matter what I did, I couldn't do it for some reason. Can't be in the SeriesListCtrl because it isn't activated until its opened which by then is too late. I figured SeriesListState.show would work but still didn't, probably cause the code to change the $state is in a .run() ? idunno
